cms/static/coffee/spec/main.coffee

```diff
@@ -27,9 +27,9 @@ requirejs.config({
         "moment": "xmodule_js/common_static/js/vendor/moment.min",
         "moment-with-locales": "xmodule_js/common_static/js/vendor/moment-with-locales.min",
         "text": "xmodule_js/common_static/js/vendor/requirejs/text",
-        "underscore": "xmodule_js/common_static/common/js/vendor/underscore",
-        "underscore.string": "xmodule_js/common_static/common/js/vendor/underscore.string",
-        "backbone": "xmodule_js/common_static/common/js/vendor/backbone",
+        "underscore": "common/js/vendor/underscore",
+        "underscore.string": "common/js/vendor/underscore.string",
+        "backbone": "common/js/vendor/backbone",
         "backbone.associations": "xmodule_js/common_static/js/vendor/backbone-associations-min",
```

Comment:
@andy-armstrong The paths are relative to the relevant `static` directory of suite. We do have symlinks to `edx-ui-toolkit`, `edx-pattern-library` and `common` in these directories and so the paths could be changed easily.

I noticed today that Karma has a `proxy` option which can be used to rewrite URLs. We can use that to get rid of these references too. But I think that will let us use the full paths instead of the symlinks too (and thus get rid of the path rewriting that is being done for coverage). So it would make sense to do all that work in one go.